lib/v2/njxzc/home.js

```diff
@@ -0,0 +1,101 @@
+// 导入got库，该库用来请求网页数据
+const got = require('@/utils/got');
+// 导入cheerio库，该库用来解析网页数据
+const cheerio = require('cheerio');
+// 导入parseDate函数，该函数用于日期处理
+const { parseDate } = require('@/utils/parse-date');
+// 导入timezone库，该库用于时区处理
+const timezone = require('@/utils/timezone');
+
+// 通知通告的url链接
+const url = 'https://www.njxzc.edu.cn/89/list.htm';
+// 南京晓庄学院的首页网址
+const host = 'https://www.njxzc.edu.cn';
+
+module.exports = async (ctx) => {
+    // 发起Http请求，获取网页数据
+    const response = await got({
+        method: 'get',
+        url,
+    });
```

Comment:
got request can be simplified
```suggestion
    const response = await got(url);
```